mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -222,6 +228,7 @@ CLASS ahm net/minecraft/entity/Entity
 	METHOD b onPlayerCollision (Larz;)V
 	METHOD b interact (Larz;Lagg;)Z
 		ARG 1 player
+	METHOD b vec3dHorizontalLengthSq (Lcnl;)D
```

Comment:
Does horizontal length squared mean just `x*x + z*z`? Moreover, this method name sounds a bit weird

Reply:
> Does horizontal length squared mean just `x*x + z*z`?

Yes, `return vec3d.x * vec3d.x + vec3d.z * vec3d.z;` is the entirety of the method body. Also, it's a static method.